ejb/WFLY-4699-expose-ejb3-deployment-info.adoc

```diff
@@ -0,0 +1,517 @@
+= Expose EJB3 Deployment Information as Runtime Resources
+:author:            Cheng Fang
+:email:             cfang@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-4699 (Expose EJB3 deployment information at runtime)
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/EAP7-435 (Expose EJB3 deployment information at runtime)
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:tborgato@redhat.com[Tommaso Borgato]
+* mailto:szhantem@redhat.com[Sultan Zhantemirov]
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE.
+// Discuss with QE during the Kickoff state to decide this
+[x] Engineering
+
+[ ] QE
+
+=== Affected Projects or Components
+
+* ejb3
+
+=== Other Interested Projects
+
+== Requirements
+
+Users should be able to view deployment information for an EJB as runtime resources via CLI or admin console.
```

Comment:
@chengfang could you please write some explicit sentence about admin console: my understanding is that no modifications to the admin console are in the scope of this RFE; if info pop up there is because of automatism that is already in place;

Reply:
Sure, I just added that in the nice-to-have section below.

Reply:
@chengfang but nothing explicitly states that admin console work is outside the scope of this RFE and what we would see there is the result of pre-existing mechanics; we would like something stating this explicitly